libevmasm/EVMAssemblyStack.h

```diff
@@ -59,6 +59,9 @@ class EVMAssemblyStack: public AbstractAssemblyStack
 	virtual std::string const* sourceMapping(std::string const& _contractName) const override;
 	virtual std::string const* runtimeSourceMapping(std::string const& _contractName) const override;
 
+	virtual Json ethdebug(std::string const& _contractName, bool _runtime) const override;
+	virtual Json ethdebug() const override;
```

Comment:
```suggestion
	Json ethdebug(std::string const& _contractName, bool _runtime) const override;
	Json ethdebug() const override;
```
no need to declare virtual here although it is done for the other functions in the header, the predominantly used style throughout the codebase is without

Reply:
yes, you're totally right! the thing is that exactly in this file all the methods are declared like this - but yeah, we definitely should clean that up

Reply:
However, looks like that I introduced it like that 😅 - I will create a separate PR

Reply:
See https://github.com/ethereum/solidity/pull/15697